src/qgis_stac/conf.py

```diff
@@ -65,10 +65,13 @@ def from_qgs_settings(
         :returns: Connection settings object
         :rtype: ConnectionSettings
         """
-        created_date = datetime.datetime.strptime(
-            settings.value("created_date"),
-            "%Y-%m-%dT%H:%M:%S.%fZ"
-        )
+        try:
+            created_date = datetime.datetime.strptime(
```

Comment:
@Fanevanjanahary instead on adding another try block can put the created_date logic inside the below try block. so we can have one try block for auth_cfg and created_date properties.